phylogenetic/build-configs/ci/config.yaml

```diff
@@ -1,2 +1,9 @@
-custom_rules:
-  - build-configs/ci/copy_example_data.smk
+inputs:
+  - name: ncbi
+    metadata: "example_data/metadata.tsv"
+    sequences: "example_data/sequences.fasta"
```

Comment:
Just flagging this as an example of spiking in local data, and bypassing download entirely